bin/main.rs

```diff
@@ -171,9 +189,12 @@ async fn main() -> anyhow::Result<()> {
 
     let store = nats::ensure_kv_bucket(&context, args.state_bucket, 1).await?;
 
-    // TODO: use the storage engine
     let state_storage = NatsKvStore::new(store);
 
+    let store = nats::ensure_kv_bucket(&context, args.manifest_bucket, 1).await?;
```

Comment:
Not sure if this diff is weird but it looks like this line is duplicated

Reply:
This is what is in code (which is correct)

```rust
let store = nats::ensure_kv_bucket(&context, args.state_bucket, 1).await?;

let state_storage = NatsKvStore::new(store);

let store = nats::ensure_kv_bucket(&context, args.manifest_bucket, 1).await?;

let manifest_storage = NatsKvStore::new(store);
```